gwctl/pkg/resourcediscovery/node_resource.go

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Lets move the contents, as is, to `printer/common.go`. I too have an upcoming need to create a file called `printer/common.go` and these should also naturally fit there. These seem to be mostly concerned with the presentation of data (hence inside printer) and not necessarily with how it's modelled